CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 * Added
   * Added ApplicationStatusOverlay
   * Added ApplicationStatusOverlayProvider to ApplicationBase, ApplicationNavigation, ModalManager and SlidePanelManager
+  * Added EventEmitter
```

Comment:
This has snuck into a released section over time

Reply:
updated.